core/dbt/parser/unit_tests.py

```diff
@@ -203,6 +206,30 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock) -> None:
         self.schema_parser = schema_parser
         self.yaml = yaml
 
+    def _get_seed_name_from_ref(self, ref: str) -> str:
+        """Extract seed name from ref string."""
+        return py_extract_from_source("{{ " + ref + " }}")["refs"][0]["name"]
+
+    def _load_rows_from_seed(self, seed_name: str) -> List[Dict[str, Any]]:
+        """Read rows from seed file on disk if not specified in YAML config. If seed file doesn't exist, return empty list."""
+        rows: List[Dict[str, Any]] = []
+
+        package_name = self.project.project_name
+
+        seed_node = self.manifest.ref_lookup.find(seed_name, package_name, None, self.manifest)
+
+        if not seed_node:
+            raise ParsingError(
+                f"Unable to find seed '{package_name}.{seed_name}' for unit tests in directories: {self.project.seed_paths}"
```

Comment:
this may be misleading because the seed (if found) could come from seed directories from packages as well (e.g. seed_nodes.original_file_path)